src/screens/ProfileScreen/elements/ProfileHeader.js

```diff
@@ -3,6 +3,7 @@ import * as React from 'react';
 import {StyleSheet, Text, View, TouchableOpacity, Platform} from 'react-native';
 import {useNavigation} from '@react-navigation/native';
 
+// TODO: Garry, need icon props fill color
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The TODO comment indicates that the `ArrowLeftIcon`, `SettingIcon`, and `ShareIcon` components need to accept a fill color as a prop. However, it seems like this hasn't been implemented yet. This could lead to inconsistencies in your icon colors across different screens or states. Consider passing the fill color as a prop to these SVG components.

```diff
- import ArrowLeftIcon from '../../../assets/icons/images/arrow-left.svg';
- import SettingIcon from '../../../assets/icons/images/setting.svg';
- import ShareIcon from '../../../assets/icons/Ic_share';

+ import ArrowLeftIcon from '../../../assets/icons/images/arrow-left.svg';
+ ArrowLeftIcon.defaultProps = { fill: 'defaultColor' };
+ import SettingIcon from '../../../assets/icons/images/setting.svg';
+ SettingIcon.defaultProps = { fill: 'defaultColor' };
+ import ShareIcon from '../../../assets/icons/Ic_share';
+ ShareIcon.defaultProps = { fill: 'defaultColor' };
```
Please replace `'defaultColor'` with the actual default color you want to use for these icons.



<!-- This is an auto-generated comment by OSS CodeRabbit -->